backend/.env.example

```
@@ -0,0 +1,21 @@
+# Plaintext in .env; the server hashes it with bcrypt in memory at startup and never stores plaintext after load.
+ADMIN_PASSWORD=your-admin-password
+
+# Required for JWT signing in production (tests set a default in conftest).
+JWT_SECRET=your-very-long-random-secret-value-at-least-32-bytes
+# Keep short because logout revocation is in-memory for this single-server app.
+JWT_EXPIRE_MINUTES=15
+# Refresh token lifetime in days for silent access-token renewal.
+ADMIN_REFRESH_EXPIRE_DAYS=7
+# HTTPS production: set to true so the refresh cookie is not sent over plain HTTP.
+# Local dev over http://localhost typically keeps this false.
+ADMIN_REFRESH_COOKIE_SECURE=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_REFRESH_COOKIE_SECURE key should go before the ADMIN_REFRESH_EXPIRE_DAYS key

(UnorderedKey)